components/locale_core/src/preferences/extensions/unicode/keywords/collation.rs

```diff
@@ -40,3 +40,26 @@ enum_keyword!(
         /// Pinyin ordering for Latin, zhuyin order for Bopomofo and CJK characters (used in Chinese)
         ("zhuyin" => Zhuyin),
 }, "co");
+
+impl CollationType {
+    #[doc(hidden)]
+    pub fn as_str(self) -> &'static str {
```

Comment:
@zbraniecki can this be generated? It's not trivial because of nesting, but I need this to create an allocation-free `DataMarkerAttributes`.

Reply:
Added in #5788